lambdas/process-channel/index.py

```diff
@@ -111,7 +111,47 @@ def parse_rss(text):
             channel_properties["skipDays"] = [day.text for day in root.findall('./channel/skipDays/day')]
     # Return channel element values and channel items
     return channel_properties, items
- 
+
+
+#Parse XML content of Atom feed (.atom RSS feeds)
+def atom_parser(text):
+    channel_properties = {}
+    items = []
+
+    root = ET.fromstring(text)
+    atom_namespace = root.tag.split('}')[0][1:]
```

Comment:
The atom namespace won't every change, it's always `http://www.w3.org/2005/Atom`, not sure you need to worry about parsing it out here. Could be that checking for the namespace on the root might be a way to determine if it's atom or RSS, though.